elastic/kibana
# [Control group] Open "add control editor" with last used width and grow settings

### test instructions
1. run kibana with `yarn start --run-examples`
2. navigate to the "Controls" developer example
3. Click "Add new data-control" button. Notice how the menu sets "width" to medium and selects "grow". 
4. Add new control, setting "width" to small and unselected "grow". Click "Save and close".
5. Click "Add new data-control" button. Notice how the menu sets "width" and "grow" to last used values.
